engine/access/rpc/backend/backend_scripts.go

```diff
@@ -112,16 +121,17 @@ func (b *backendScripts) executeScriptOnExecutionNode(
 	for _, execNode := range execNodes {
 		result, err := b.tryExecuteScript(ctx, execNode, execReq)
 		if err == nil {
-			b.log.Debug().
+			logger.Debug().
 				Str("execution_node", execNode.String()).
-				Hex("block_id", blockID[:]).
-				Str("script", string(script)).
-				Msg("Successfully executed script")
+				Msg("successfully executed script")
 			return result, nil
 		}
 		errors = multierror.Append(errors, err)
 	}
-	return nil, errors.ErrorOrNil()
+
+	err = errors.ErrorOrNil()
+	logger.Err(err).Msg("failed to execute script")
```

Comment:
Does this error include all the exe nodes we tried? if not should we add that to this log message?

Reply:
yeah it does include the exe node name as part of the message (https://github.com/onflow/flow-go/pull/452/files#diff-d4f96095c328265258f0b505e475292ed7d9460a4591033eae31271798ad74caR140)